frontend/src/app/modules/fields/edit/field-types/multi-select-edit-field.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="textarea-wrapper">
+<div>
```

Comment:
We can probably remove the whole div then, right?

Reply:
Hi @b12f ,
Thanks for your feedback.
I'm not sure, I think it is better if @oliverguenther could take a look at it. This is why I assigned the review to him.